plugin/effect/src/index.ts

```diff
@@ -1,21 +1,20 @@
-import { Effect } from "effect";
+import * as Effect from "effect/Effect";
```

Comment:
alias imports are better for library code, as they cause less issues with bundlers